Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -43,20 +53,34 @@ FieldProbe::FieldProbe (std::string rd_name)
     pp_amr.query("max_level", nLevel);
     nLevel += 1;
 
+    /* Obtain input data from parsing inputs file.
+     * For the case of a single particle:
+     *     Define x, y, and z of particle
+     *     Define whether or not to integrate fields
+     * For the case of a line detector:
+     *     Define x, y, and z of end of line point 1
+     *     Define x, y, and z of end of line point 2
+     *     Define resoliton to determine number of particles
+     *     Define whether ot not to integrate fields
+     * For the case of a plane detector:
+     *     Define a vector normal to the detector plane
+     *     Define a vector in the "up" direction of the plane
+     *     Define the size of the plane (width of half square)
+     *     Define resoliton to determine number of particles
+     *     Define whether ot not to integrate fields
+     */
     amrex::ParmParse pp_rd_name(rd_name);
     getWithParser(pp_rd_name, "x_probe", x_probe);
-#if (AMREX_SPACEDIM == 3)
     getWithParser(pp_rd_name, "y_probe", y_probe);
-#endif
     getWithParser(pp_rd_name, "z_probe", z_probe);
-
+    pp_rd_name.query("integrate", m_field_probe_integrate);
     pp_rd_name.query("raw_fields", raw_fields);
-
     pp_rd_name.query("interp_order", interp_order);
 
     AMREX_ALWAYS_ASSERT_WITH_MESSAGE(interp_order <= WarpX::nox ,
-                                     "Field probe interp_order should be lower than algo.particle_shape");
+                                     "Field probe interp_order should be lower or equal than algo.particle_shape");
```

Comment:
"should be less than or equal to"
```suggestion
                                     "Field probe interp_order should be less than or equal to algo.particle_shape");
```